examples/SC_noise_capture.py

```diff
@@ -0,0 +1,96 @@
+import pyspecdata as ps
+import numpy as np
+import os
+from numpy import r_
+import SpinCore_pp as sc
+from datetime import datetime
+
+my_exp_type = "ODNP_NMR_comp/noise_tests"
+assert os.path.exists(ps.getDATADIR(exp_type=my_exp_type))
+config_dict = sc.configuration("active.ini")
+nScans = 100
+# {{{ add file saving parameers to config dict
+config_dict["chemical"] = (
+    config_dict["chemical"] + "_" + str(config_dict["SW_kHz"]) + "kHz"
+)
+config_dict["type"] = "noise"
+config_dict["date"] = datetime.now().strftime("%y%m%d")
+config_dict["noise_counter"] += 1
+# }}}
+# {{{ SpinCore settings - these don't change
+tx_phases = r_[0.0, 90.0, 180.0, 270.0]
+(
+    nPoints,
+    config_dict["SW_kHz"],
+    config_dict["acq_time_ms"],
+) = sc.get_integer_sampling_intervals(
+    SW_kHz=config_dict["SW_kHz"],
+    time_per_segment_ms=config_dict["acq_time_ms"],
+)
+tau_us = 1000
+data_length = 2 * nPoints * 1 * 1  # assume nEchoes and nPhaseSteps = 1
+RX_nScans = 1
+# }}}
+# {{{ Acquire data
+for x in range(nScans):
+    # {{{ configure SpinCore
+    sc.configureTX(
+        config_dict["adc_offset"],
+        config_dict["carrierFreq_MHz"],
+        tx_phases,
+        config_dict["amplitude"],
+        nPoints,
+    )
+    acq_time = sc.configureRX(
+        config_dict["SW_kHz"],
+        nPoints,
+        RX_nScans,
+        1,  # assume nEchoes = 1
+        1,  # assume nPhaseSteps = 1
+    )
+    sc.init_ppg()
+    # }}}
+    # {{{ ppg to generate the SpinCore data
+    sc.load(
+        [
+            ("marker", "start", 1),
+            ("phase_reset", 1),
+            ("delay", tau_us),
+            ("acquire", acq_time),
+            ("delay", 1e4),  # short rep delay
+            ("jumpto", "start"),
```

Comment:
unsure if all delays are needed but figure since this is how I've been acquiring I stick to that